<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginCreatorBase</class>
 <widget class="QMainWindow" name="PluginCreatorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>456</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="run">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="Ms::QmlEdit" name="textEdit"/>
      <widget class="QPlainTextEdit" name="log"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Close Plugin Creator</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload current plugin source</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ms::QmlEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qmledit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
